home_assistant_datasets/tool/leaderboard/prebuild.py

```diff
@@ -56,6 +56,8 @@ def write_eval_output(cmds: list[str], output_file: pathlib.Path) -> None:
 
     def build_report(eval_report: EvalReport) -> None:
         print(f"Generating report for outputs in {eval_report.directory}")
+        if eval_report.directory.parent.name == 'automations':
```

Comment:
I see the problem, we should instead fix https://github.com/allenporter/home-assistant-datasets/blob/1edcee89f46de45346eedeec2112d05d47c75dc0/home_assistant_datasets/tool/leaderboard/config.py#L54 so that `eval_reports` doesn't iterate over this. 

I think we should make a new variable `PREBUILD_DATASETS` with the set of datasets in the overall set that excludes automations. That way we don't add it to the list then exclude it here, we just don't add it to the start to begin with.